implementation-script/main.py

```diff
@@ -38,7 +38,7 @@
 from openpyxl.styles import colors
 from openpyxl.cell import Cell
 import gdown
-
+import jwt
```

Comment:
@praveenKDass Please test the change regarding implementation with and without programs. Also test the dynamic scoping logic

Reply:
Tested with and without program logic @VISHNUDAS-tunerlabs